lib/src/resources/movies_repository.dart

```diff
@@ -3,5 +3,7 @@ import '../model/movies_model.dart';
 
 class MoviesRepository {
```

Comment:
This repository should also be implementing an interface

Reply:
Done